scripts/version-plugin.ts

```diff
@@ -0,0 +1,21 @@
+import { execSync } from 'child_process';
+import { writeFile } from 'fs';
+import path from 'path';
+
+function recordVersion() {
```

Comment:
So do I understand correctly that this will run on every reload effectively helping us to not to forget the version change? I wonder maybe we should add some Github check whether the `public/version.json` file was changed so it is always guaranteed that new PR changes the version.

Reply:
Also on second thought, suppose the scenario is:
I make some changes -> reload changes the version (records the last commit) -> I commit changes -> I push changes 
This scenario produces the wrong version as the last recorded commit is actually the second last commit. It holds true even for this PR. Last commit hash is `fbe3f18` however last recorded version commit in `public/version.json` is `97c0788`.  We should think this through more carefully as this might provide a lot of false positives (eg. OSIM shows it runs on commit N-1 but actually it runs on commit N.